presto-main/src/test/java/com/facebook/presto/block/AbstractTestBlock.java

```diff
@@ -199,7 +203,12 @@ else if (type == MethodHandle.class) {
 
     private <T> void assertBlockPositions(Block block, Supplier<BlockBuilder> newBlockBuilder, T[] expectedValues)
     {
-        assertEquals(block.getPositionCount(), expectedValues.length);
+        try {
```

Comment:
I don't understand this change. Would you explain?

Reply:
@mbasmanova Oh I'm sorry, this was for debugging purpose. I've removed the change.